Makefile.core.mk

```diff
@@ -354,6 +354,10 @@ update-golden: refresh-goldens
 
 gen: mod-download-go go-gen mirror-licenses format update-crds operator-proto sync-configs-from-istiod gen-kustomize update-golden ## Update all generated code.
 
+update-operator-data-snapshot:
```

Comment:
Should this be added as a dependency for another target?

Reply:
No, the  point is that its not updated every change, so we don't cause 100s of tests to update when we change a small file in manifests/

Should this run ./operator/scripts/run_update_golden_snapshots.sh?

Reply:
We could. I can't recall what I was doing, but I had a problem with `make gen-go` or a similar command failing recently. I tracked it down to the tar.gz. I don't think adding a target makes it more easily fixable by someone. I didn't notice the run_update_golden_snapshots.sh file. I'll just close out the PR.